en/guide/directory-structure.md

```diff
@@ -41,23 +41,23 @@ _This directory cannot be renamed._
 
 ### The Plugins Directory
 
-The `plugins` directory contains your Javascript plugins that you want to run before instantiating the root vue.js application.
+The `plugins` directory contains your Javascript plugins that you want to run before instantiating the root Vue.js Application.
 
 [More documentation about Plugins integration](/guide/plugins)
 
 ### The Static Directory
 
-The `static` directory contains your static files. Each file inside this directory is mapped to /.
+The `static` directory contains your static files. Each file inside this directory is mapped to `/`.
 
-**Example:** /static/robots.txt is mapped as /robots.txt
+**Example:** `/static/robots.txt` is mapped as `/robots.txt`
 
 _This directory cannot be renamed._
 
 [More documentation about Static integration](/guide/assets#static)
 
 ### The Store Directory
 
-The `store` directory contains your [Vuex Store](http://vuex.vuejs.org) files. The Vuex Store option is implemented in the Nuxt.js framework. Creating an `index.js` file in this directory enables the option in the framework automatically.
+The `store` directory contains your [Vuex Store](http://vuex.vuejs.org/en/) files. The Vuex Store option is implemented in the Nuxt.js framework. Creating an `index.js` file in this directory enables the option in the framework automatically.
```

Comment:
Because they are no necessity to link the « all language » page for « english » documentation.